dags/hivemind_etl_helpers/tests/integration/test_gdrive_loader.py

```diff
@@ -130,7 +131,7 @@ def test_load_by_file_id(self, mock_load_data):
     @patch.object(GoogleDriveReader, "load_data")
     def test_load_from_folders_exception(self, mock_reader):
         mock_loader = Mock()
```

Comment:
There's no need for creating a variable `mock_loader` here. Please consider removing it.